backend/routes/bookmark.js

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
Can you please remove the empty line in line 1.

Reply:
yeah sure